chain-impl-mockchain/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "chain-impl-mockchain"
+version = "0.1.0"
+authors = [ "Nicolas Di Prima <nicolas.diprima@iohk.io>"
+          , "Vincent Hanquez <vincent.hanquez@iohk.io>"
+          , "Eelco Dolstra <edolstra@gmail.com>"
+          , "Mikhail Zabaluev <mikhail.zabaluev@gmail.com>"
+          , "Alexander Vershilov <alexander.vershilov@gmail.com>"
+          ]
+edition = "2018"
+
+[dependencies]
+serde = "^1.0"
+serde_derive = "^1.0"
+bincode = "^1.0.1"
+cryptoxide = "*"
+quickcheck = { version = "0.7", optional = true }
+chain-core = { path = "../chain-core", features=["property-test-api"]}
```

Comment:
this feature should only be enabled in the `dev-dependencies`

Reply:
Done